content/en/04-tutorials/03-integrations/03-otel.md

```diff
@@ -86,50 +121,39 @@ df = df.groupby(['pod', 'service', 'req_path']).agg(
 df.requests_per_s = df.throughput / 10
 ```
 
+### Exporting
 
-
-## Exporting
-
-To export the data, you’ll call `px.export` with the DataFrame as the first argument and the export target `px.otel.Data` as the second argument. 
-
+To export the data, you’ll call `px.export` with the DataFrame as the first argument and the export target `px.otel.Data` as the second argument.
 
 ```python
 px.export(df, px.otel.Data(...))
 ```
 
-
 The export target (`px.otel.Data`) describes which columns to use for the corresponding OpenTelemetry fields. You specify a column using the same syntax as in a regular query: `df.column_name` or `df[‘column_name’]`. The columns must reference a column available in the `df` argument or the PxL compiler will throw an error
 
-
-## Specifying a Collector Endpoint and Authentication
+### Specifying a Collector Endpoint and Authentication
 
 The PxL OpenTelemetry exporter needs to talk with a collector. You must specify this information via the `endpoint` parameter:
 
-
 ```python
 endpoint=px.otel.Endpoint(
-  url='0.0.0.0:55690',
-  headers={
-    'api-key': '12345',
-  }
+  url='otel-collector.default.svc.cluster.local:4317',
+  insecure=True
 ),
 ```
 
-
-The endpoint url must be an OpenTelemetry grpc endpoint and must be secured with SSL. Don’t specify a protocol prefix. Optionally, you can also specify the headers passed to the endpoint. Some OpenTelemetry collector providers look for authentication tokens or api keys in the connection context. The headers field is where you can add this information.
+The endpoint url must be an OpenTelemetry grpc endpoint. Don’t specify a protocol prefix. Optionally, you can also specify the headers passed to the endpoint. Some OpenTelemetry collector providers look for authentication tokens or api keys in the connection context. The headers field is where you can add this information.
 
 Note that if you’re writing a [plugin script](/reference/plugins/plugin-system), this information should be passed in from the plugin context.
```

Comment:
Change to: "this information will be passed in by the plugin context and should not be specified".